docs-new/docs/components/lifecycle-operator/deployment-flow.md

```diff
@@ -57,7 +57,7 @@ Only then are the pods bound to a node and deployed.
 If any pre-deployment evaluation or task fails,
 the `KeptnApp` issues an appropriate `*Errored` event
 and the deployment remains pending indefinitely,
-until further changes or external intervention
+until further changes or external intervention.
```

Comment:
```suggestion
until further changes or external intervention
either resolve the problem or terminate the execution.
```
The `until` clause needs a predicate so I made something up but I doubt that the content is correct.

Reply:
Not sure I understand what you mean here. In this place there was a missing dot, which I added. The sentence in general makes sense to me

Reply:
I think what Meg means is that until is usually sustained by a verb and cannot be alone "until ... something happens", I guess in your sentence it could be 'until an other change takes place'

Reply:
@odubajDT ?